src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[NUMBER_OF_BALLS];
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = new Lottery().getRandomBall();
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/147297fb-fccf-413f-98eb-4a87f9ef01fb)

Reply:
Done. Now only one ball is created in Main class.